### DISCUSSION TITLE
CC2640 1.8V-3.8V Bluetooth BLE 4.1

### DISCUSSION THREAD

**Discussion by ollimagic** [mKeRix/room-assistant#791] [General]
Hi i have a question is it possible to make these Beacons work with room-assistant?
they have diffrent bluetooth mac adresses but it seems room-assistant can only read the UUID
and that is for all the beacons the same.
Any suggestions for me?

**Comment by mKeRix** [comment]
I don't know these specific beacons, but if you don't want to use the iBeacon UUID you can use its regular MAC by setting `processIBeacon: false` under `bluetoothLowEnergy` ([docs](https://www.room-assistant.io/integrations/bluetooth-low-energy.html#settings)).

**Reply by ollimagic** [reply]
thank you so much for the help it is woking now i have one question left i have to PI Zero but they loose always after the while the connection to my home-assistant for maybe a min and the they go back online how can i avoid that it happen to my PI3 too but there i disable wifi and now they are stable but it is not possible fo my PI Zeros :-) is there any suggestion how i can avoid that?
Regards 
Oliver

**Reply by mKeRix** [reply]
That could happen for a number of reasons, so it's hard to give generic advice for it - a few things that you can try are:

- observe overall WiFi link quality on the Pi Zero and play with its placement
- make a stable member of the cluster the [preferred leader](https://www.room-assistant.io/guide/cluster.html#preferred-leaders)
- attach USB Bluetooth dongles to the Pi Zeros (and configure room-assistant to use them), so that they may use the shared internal chip for WiFi only

**Comment by ollimagic** [comment]
OK i have some more questions when i put in my RA configuration on my Home-Assitant (intigration) the cluster part, it dont show anymore my Bluetooth devices. My Pi Zero gont get any Bluetooth Lowenergy Devicey only Classic ( i bought a dongle and will try it tomorrow.)

cluster:
  autoDiscovery: false
  timeout: 120
  quorum: 2
  weight: 1
  port: 6425
  peerAddresses:
    - 192.168.xxx.118:6425 #Hassio Arbeitszimmer
    - 192.168.xxx.193:6425 #BillardKüche
    - 192.168.xxx.195:6425 #WohnTerasse
    - 192.168.xxx.189:6425 #Schlafzimmer
    - 192.168.xxx.190:6425 #Eingangsbereich
    - 192.168.xxx.187:6425 #Keller
global:
  instanceName: Schlafzimmer
  integrations:
    - homeAssistant
    - bluetoothClassic
    - shell
    - bluetoothLowEnergy
homeAssistant:
  mqttUrl: 'mqtt://192.xxx.xx.118:1883'
  mqttOptions:
    username: xxxxt
    password: xxxxxx
  discoveryPrefix: homeassistant
shell:
  sensors:
    - name: Schlafzimmer Wifi Strength
      command: 'iwconfig wlan0 | grep -i quality'
      regex: 'Signal level=(-?[0-9]+) dBm'
      cron: '* * * * *'
      icon: mdi:wifi
      unitOfMeasurement: dBm
      deviceClass: signal_strength
bluetoothLowEnergy:
  hciDeviceId: 0
  timeout: 60 
  processIBeacon: false
  allowlist:
   - cxxxx3f92cc7
   - cxxxx3f91c26
   - cxxxx3f927a4
   - exxxx6bd7f5f
  tagOverrides:
    cxxxx3f92cc7:
     name: Beacon1
     measuredPower: -61
     batteryMask: 0x0000FF00
    cxxxx3f91c26:
     name: Beacon2
     measuredPower: -61
     batteryMask: 0x0000FF00
    cxxxx3f927a4:
     name: Beacon3
     measuredPower: -61
     batteryMask: 0x0000FF00
    exxxx6bd7f5f:
     name: OlliWatchLE
     measuredPower: -61
     batteryMask: 0x0000FF00
bluetoothClassic:
  hciDeviceId: 0
  interval: 6
  scanTimeLimit: 2.5
  timeoutCycles: 2.5
  inquireFromStart: true
  minRssi:
    '18:xx:xx:5c:14:a8': -25
    '34:xx:xx:0d:03:79': -25
    'e0:xx:xx:bd:7f:5f': -25
  addresses:
    - 'e0:aa:96:bd:7f:5f' #  Watch
    - '18:xx:xx5c:14:a8' #  1Handy
    - '34:xx:xx:0d:03:79' #2elefon
  entityOverrides:
     '18:xx:xx:5c:14:a8':
        id: OllisPhone
        name: 1Phone
     '34:xx:xx:0d:03:79':
        id: ChristofsPhone
        name: ChristofsPhone
     'e0:xx:xx:bd:7f:5f':
        id: WatchClassic
        name:  WatchClassic

can you tell me what is wrong in my config cluster makes everywhere trouble Low energy only on PI Zero Home-Assitant Iintigration is on a PI4
Danke für die Hilfe

**Reply by ollimagic** [reply]
made it work with a second dongle no more problems with going on and offline
